src/index.html

```diff
@@ -4,7 +4,7 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>PokeAPI</title>
```

Comment:
Thanks a lot for changing the name of the page! However, I think we should change it to something a little more appealing like "Pokedex" or something along those lines.

Reply:
Thanks for the suggestion. I'll implement that